packages/README.md

```diff
@@ -4,6 +4,8 @@
 
 @osmosis-labs/stores - contains Mobx stores for accounts and queries, price data, historical data (imperator), common UI config (creating pool, swapping, managing liquidity) and the GAMM currency registrar
 
+@osmosis-labs/osmojs - Contains libraries for interacting with the Osmosis chain
```

Comment:
Is `osmojs` a fitting name? Isn't this like `types` or something since it has the proto encoders/decoders?

Regardless- IMO this description leaves a lot of room for improvement.

Reply:
> Is osmojs a fitting name? Isn't this like types or something since it has the proto encoders/decoders?

I haven't decided on the name. Instead of osmojs, I'll name it `proto-encoder` as a placeholder. For sure, there must be a better name for this. I'm open to any suggestions.

This is a library to generate the blockchain encoders/decoders plus amino types. 

> Regardless- IMO this description leaves a lot of room for improvement.

For sure, I'll be improving this